docs/UserGuide.md

```diff
@@ -25,15 +25,15 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+* `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+* `delete 3` : Deletes the 3rd contact shown in the current list.
 
-   * `clear` : Deletes all contacts.
+* `clear` : Deletes all contacts.
 
-   * `exit` : Exits the app.
+* `exit` : Exits the app.
```

Comment:
One minor suggestion here is that by indenting the asterisk forward, the list is no longer nested in the number list (initially from 1 to 6), and now the last point (i.e., Refer to the [Features]...) is renumbered from 6 to 1. 
So I think perhaps it's better to keep the indentation as it was in the original version?

Reply:
Thanks for pointing this out. I had copied this from another source, which caused this formatting issue. Will fix it now.